mage_ai/orchestration/db/models/schedules.py

```diff
@@ -263,20 +265,32 @@ def create_or_update_batch(self, trigger_configs: List[Trigger]):
                 traceback.print_exc()
                 continue
 
+            last_enabled_at = trigger_config.last_enabled_at
+            if trigger_config.status == ScheduleStatus.ACTIVE and \
+                    trigger_config.last_enabled_at is None:
+                last_enabled_at = datetime.now(tz=pytz.UTC)
+                trigger_config.last_enabled_at = last_enabled_at
+                add_or_update_trigger_for_pipeline_and_persist(
```

Comment:
This was added for the case where a trigger was created via code and that trigger had an `active` status, but no `last_enabled_at` property. This would automatically update the `triggers.yaml` file with a `last_enabled_at` property with the current datetime as its value.

Without this line of code, the `last_enabled_at` property is still updated in the metadata db, but not written to the `triggers.yaml` config file, causing the trigger in the metadata db to be inconsistent with the trigger via code in the `triggers.yaml` file.